README.md

```diff
@@ -1,31 +1,49 @@
-# curriculum-databases-projects-template
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ernest Portfolio
+
+> This is my portfolio project
+
+
+## Built With
+
+- Major languages: HTML and CSS
+- Technologies used: Linter
+
+
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
 
 ## Getting Started
 
-This repository includes files with plain SQL that can be used to recreate a database:
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] please update `Getting Started` section of your readme to make it look professional & to describe how to setup & start your project locally